### PR TITLE
[Amplitude] Tracking data source created

### PR DESCRIPTION
**Pull request by lasryaric**
## Description

[Amplitude] Tracking data source created

### CODE REVIEW

File: front/pages/api/w/[wId]/data_sources/index.ts [hunk 1]
```diff
@@ -170,6 +171,11 @@ async function handler(
         editedByUserId: user.id,
       });
 
+      const dataSourceType = await getDataSource(auth, ds.name);
```
Comment (flvndvd) [inline front/pages/api/w/[wId]/data_sources/index.ts:174]: You have `ds` just above which should cover your needs, right?

File: front/pages/api/w/[wId]/data_sources/managed.ts [hunk 2]
```diff
@@ -428,6 +430,10 @@ async function handler(
         connectorId: connectorsRes.value.id,
         connectorProvider: provider,
       });
+      const dataSourceType = await getDataSource(auth, dataSource.name);
```
Comment (flvndvd) [inline front/pages/api/w/[wId]/data_sources/managed.ts:433]: Same here why not using `dataSource` from above?

File: front/pages/api/w/[wId]/data_sources/index.ts (hunk 1 above)
Comment (lasryaric) [reply front/pages/api/w/[wId]/data_sources/index.ts:174]: @flvndvd Amplitude code should only use Types and Authenticator, no db model, hence this code.
Was already discussed with spolu, and I agree.

File: front/pages/api/w/[wId]/data_sources/managed.ts (hunk 2 above)
Comment (lasryaric) [reply front/pages/api/w/[wId]/data_sources/managed.ts:433]: Same here.

**Comment by lasryaric** [comment]
Merging because I answered your comments and I need to move forward with another PR that relies on it.

File: front/pages/api/w/[wId]/data_sources/index.ts (hunk 1 above)
Comment (flvndvd) [reply front/pages/api/w/[wId]/data_sources/index.ts:174]: I mean we could definitely derive the type from the db model, right?

**Comment by lasryaric** [comment]
Oh ok. I agree but I think we should move towards a central place for
rendering types, and that’s what have been done with subscriptions and some
other models, plus it’s happening only at creation time, so I don’t really
see any reason to optimize it.

**Comment by spolu** [comment]
As discussed can we rename back -> backend and use UserTypeWithWorkspace instead of the user ModelId? :pray:

That means we will want to call `trackUserMemberships` from `pages/api/user/index.ts` (which is called by the hook used by the AppLayout). This a few more calls than doing so from login, but I think it's fine?

**Comment by lasryaric** [comment]
@spolu for sure, just populating the data because the sales team seems to really need it, and I will do the clean up right after 👍

**Comment by lasryaric** [comment]
@spolu on it.

**Pull request merged**